### PR TITLE
fix: Use latest sbt 0.13 for java8+

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?
- Convert the system tests to use typescript
- Upgrade from `sbt`` 0.13.13` to `0.13.17` so we can use java 8 + for tests and local `0.13.13` is not playing nice anymore. Minimum supported version is `0.13.17`
#### Where should the reviewer start?
Run tests locally, they should now work again

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "tslint": "5.16.0"
   },
   "dependencies": {
+    "@types/node": "12.0.4",
```
Comment (kyegupov) [inline package.json]: I think this should be 6.14.6

File: lib/index.js [hunk 2]
```diff
@@ -16,8 +16,11 @@ function inspect(root, targetFile, options) {
   if (!options) {
     options = {dev: false};
   }
-
   const targetFilePath = path.dirname(path.resolve(root, targetFile));
+  if (!fs.existsSync(targetFilePath)) {
+    throw new Error(`build.sbt not found at location: ${targetFilePath}`)
+  }
```
Comment (adrukh) [inline lib/index.js]: I've seen cases where a multi-module sbt project has no `build.sbt` file in its root folder, but we can handle it when the user runs `snyk test --package-manager=sbt --file=.` or similar. A very edgy edge case, but I'm afraid this change may break it.

Suggest to check analytics for sbt tests that didn't have the manifest file available, and apply this change only if there are 0 hits on this search.

Comment (lili2311) [reply lib/index.js]: Found 1 match for `--packageManager=sbt` in the last 14 days, no mention of --file

Comment (lili2311) [reply lib/index.js]: new PR to not block this release just for the new error 
https://github.com/snyk/snyk-sbt-plugin/compare/fix/throw-is-missing-manifest?expand=1
